CONTRIBUTING.md

```diff
@@ -16,17 +16,18 @@ An eslint plugin (with an extra CLI utility) to help ease products transitioning
 
 ## Development
 
-1. Run `node generate-replace-prop-v5 [componentName] [oldPropName] [newPropName] [referencePR]` to create the rule/test files and update from there
-2. Don't forget to document the change/rule within the readme file https://github.com/patternfly/pf-codemods/blob/main/packages/pf-codemods/README.md
-3. Run `yarn test:v5` to run tests
-4. You can also update the `test.tsx` file and run `yarn test:v5:single` to see what a consumer would see if they were to run our codemods
+1. Run `yarn generate` to start the interactive CLI for the generator
+    - It will create the rule/test files and update the README/test.tsx files as needed
+    - Select the `generic` generator if you don't see a more applicable one, it creates updates demonstrating prop removal to serve as a basic canvas for you to modify
+1. Run `yarn test:v5` to run tests
+1. You can also run `yarn test:v5:single` to see what a consumer would see if they were to run our codemods
 
 If you're having trouble writing a rule, you can:
 1. Put code into an AST explorer like https://astexplorer.net/ to inspect all the AST nodes
```

Comment:
```suggestion
1. Put code into [AST explorer](https://astexplorer.net/) to inspect all the AST nodes of the input code. You should choose "JavaScript" as the language and "acorn" as the parser
    - This will provide a realtime view of what output your codemod will produce during development
```

Or whatever setup you may be using, maybe just include what options to have set in AST Explorer.